activation/src/main/java/jakarta/activation/MimetypesFileTypeMap.java

```diff
@@ -213,7 +220,12 @@ private void loadAllResources(Vector v, String name) {
 			if (LogSupport.isLoggable())
 			    LogSupport.log("MimetypesFileTypeMap: can't load " +
 						url, sex);
-		    } finally {
+		    } catch (NoSuchElementException | ServiceConfigurationError e) {
+				if (LogSupport.isLoggable()) {
```

Comment:
the spaces here look like `tabs`. Make sure real space characters are used (no `tabs`)